querybook/webapp/ui/Popover/Popover.tsx

```diff
@@ -136,6 +136,10 @@ export const PopoverContainer = React.forwardRef<
 
             if (mainLayout === 'top') {
                 newWrapperStyle.top = box.top - contentBox.height - arrowSize;
+                // keep the popover on screen
+                if (newWrapperStyle.top < 0) {
+                    newWrapperStyle.top = box.bottom;
+                }
```

Comment:
shouldn't this be same as `mainLayout === 'bottom'`, which is `newWrapperStyle.top = box.bottom + arrowSize;`?